<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsPage</class>
 <widget class="QWidget" name="ResultsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>510</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="resultsWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="KMessageWidget" name="errorWidget">
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="messageType">
          <enum>KMessageWidget::Warning</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KMessageWidget" name="lostMessage"/>
       </item>
       <item>
        <widget class="QTabWidget" name="resultsTabWidget">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="timeLineLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
